### PR TITLE
Common instancetypes periodics stable release sync

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #https://github.com/kubevirt/common-instancetypes/issues/162

**Special notes for your reviewer**:

Depends on https://github.com/kubevirt/kubevirt/pull/11368 and https://github.com/kubevirt/ssp-operator/pull/900

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml
Comment (lyarwood) [reply github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml:47]: Yeah as discussed `common-instancetypes` `v0.4.0` hadn't been sync'd with SSP `v0.19.0` but I'm doing this manually in https://github.com/kubevirt/ssp-operator/pull/901. Now that's landing I've added SSP to the `release-0.4` job.

File: github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml:27]: nit: Can this be moved to the env section of the job config?

File: github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml:73]: it might make sense to have unique branch names here as these jobs could end up running at the same time - maybe something like `update-common-instancetypes-1.2`

Same goes for the rest of the branches.

File: github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-periodics.yaml:38]: nit: Could you stagger the crons so the jobs aren't all kicking off at the same time? The created PRs would have a knock on effect to CI with a lot of jobs being triggered at once.

**Review by brianmcarey — COMMENTED** [review]
Looks good @lyarwood  - just a couple of nits and one question on the branch names used for creating the PRs.

**Comment by lyarwood** [comment]
Thanks @brianmcarey , updated but I'm going to /hold this PR for now until the script changes land and are backported.

**Comment by lyarwood** [comment]
Thanks @brianmcarey , updated but I'm going to /hold this PR for now until the script changes land and are backported. I'll let you know once it's ready to go again.

/hold

**Comment by lyarwood** [comment]
/unhold

All of the backports have now landed so this should be good to test and hopefully land at somepoint.

**Comment by lyarwood** [comment]
@brianmcarey if you have anytime in Paris this week this should be ready finally.

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Review by brianmcarey — APPROVED** [review]
Sorry @lyarwood - only seeing the ping now. Looks good to me - just placing a hold so you can decide when you want this enabled. 

/approve 

/hold

**Comment by lyarwood** [comment]
@brianmcarey thanks and no issues, lets enable this now and see how it runs overnight.

/unhold

**Pull request merged**
